package.json

```diff
@@ -3,6 +3,9 @@
   "bugs": {
     "url": "https://github.com/userdive/agent.js/issues"
   },
+  "dependencies": {
+    "eslint-plugin-react": "7.13.0"
```

Comment:
It should be `devDependencies` here instead of `dependencies` , shouldn't it?

Reply:
oh...
I will fix it soon! thanks.